lib/reactive_table.js

```diff
@@ -500,7 +506,7 @@ Template.reactiveTable.events({
         } else {
             template.context.sortKey.set(sortFieldId);
         }
-        updateHandle.call(template.context);
+        getUpdateHandleForTemplate(template).call(template.context, template.context);
```

Comment:
This could just be `getUpdateHandlerForTemplate(template)(template.context)` - `call` is just to set what `this` will be in the function.